src/main/resources/com/mathworks/ci/MatlabBuilder/config.jelly

```diff
@@ -1,7 +1,7 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 
- <font color="red"> This build step is depricated. Consider moving to the new MATLAB build steps. Current build however, will work with restricted access.</font>   
+ <font color="red">Using this build step is not recommended and will be removed in a feature release. Use “Run MATLAB Tests” or “Run MATLAB Command” instead.</font>   
```

Comment:
@mw-hrastega May i request you to have a look at this warning message

Reply:
Looks fine to me. Thanks.